Header header

string map_id

string robot_id

geometry_msgs/Polygon polygon
